src/DZ-2/Test.java

```diff
@@ -0,0 +1,43 @@
+package geekbrains_java_algorithms_2021.DZ_2;
+
+public class Test {
+
+    public static void main(String[] args) {
+        int SIZE_ARRAY = 1_00_000;
+        ArrayImpl<Integer> bubbleArr = getRandomArrImpl(SIZE_ARRAY);
+        ArrayImpl<Integer> selectArr  = new ArrayImpl<>();
+        selectArr.size = bubbleArr.size;
+        selectArr.data = bubbleArr.data;
```

Comment:
```suggestion
        selectArr.data = Arrays.copyOf(bubbleArr.data, bubbleArr.data.length);
```
иначе мы используем один массив для всех сортировок и после первой сортировки наши данные уже будут упорядочены. Отсюда и такое быстродействие сортировки вставкой.

Аналогично нужно сделать и для другого массива